test/unit/spec/IDPDiscovery_spec.js

```diff
@@ -594,6 +594,25 @@ function (Q, OktaAuth, WidgetUtil, Okta, Util, AuthContainer, IDPDiscoveryForm,
     });
 
     Expect.describe('Device Fingerprint', function () {
+      itp(`does not compute device fingerprint if security image is off, deviceFingerprinting
+        is true and useDeviceFingerprintForSecurityImage is true)`, function () {
+        spyOn(DeviceFingerprint, 'generateDeviceFingerprint').and.callFake(function () {
```

Comment:
Should this just be spyOn? We don't need to define the fake function because we are not calling it right?